src/instruments/src/SD/Pages/Wheel/Wheel.tsx

```diff
@@ -178,114 +232,241 @@ type GearLocation = 'left' | 'center' | 'right';
 
             <GearDoorJoint x={location === 'left' ? 103.01 : 21.13} y={0} />
         </SvgGroup>
     );
 };
 
 interface LandingGearPositionIndicatorsProps extends ComponentPositionProps {
-    inTransit: boolean,
-}
-
-const LandingGearPositionIndicators = ({ x, y, inTransit }: LandingGearPositionIndicatorsProps) => (
-    <SvgGroup x={x} y={y} className="gear-lgcius">
-        <g className={`shape gear-lgciu-1 color-${!inTransit ? 'green' : 'red'}`}>
-            <polygon points="0,0 22,0 22,29" />
-            <path d="m 6 0 v 8" />
-            <path d="m 11 0 v 14" />
-            <path d="m 16.5 0 v 22" />
-        </g>
-
-        <g className={`shape gear-lgciu-2 color-${!inTransit ? 'green' : 'red'}`}>
-            <polygon points="30,0 52,0 30,29" />
-            <path d="m 36 0 v 22" />
-            <path d="m 41 0 v 14" />
-            <path d="m 46 0 v 8" />
-        </g>
-        <g className="shape gear-failed-lgicu-1">
-            <path d="m 0 0 h 22" />
-
-            <text className="gear-failed-lgciu-xx" x={13} y={16} fontSize={17}>XX</text>
-        </g>
-
-        <g className="shape gear-failed-lgicu-2">
-            <path d="m 30 0 h 22" />
-
-            <text className="gear-failed-lgciu-xx" x={39} y={16} fontSize={17}>XX</text>
-        </g>
-    </SvgGroup>
-);
-
-interface GearProps extends ComponentPositionProps {
-    location: GearLocation
+    location: GearLocation,
+    lgciu1DiscreteWord1: Arinc429Word,
+    lgciu2DiscreteWord1: Arinc429Word,
+    lgciu1DiscreteWord3: Arinc429Word,
+    lgciu2DiscreteWord3: Arinc429Word,
 }
 
-const Gear = ({ x, y, location }: GearProps) => {
-    const [landingGearPosition] = useSimVar(`GEAR ${location.toUpperCase()} POSITION`, 'Percent Over 100', maxStaleness);
-
-    const [status, setStatus] = useState({
-        inTransit: false,
-        positionIndicatorVisible: false,
-    });
-
-    useEffect(() => {
-        const isUp = landingGearPosition === 0;
-        const isDown = landingGearPosition === 1;
+const LandingGearPositionIndicators = ({ x, y, location, lgciu1DiscreteWord1, lgciu2DiscreteWord1, lgciu1DiscreteWord3, lgciu2DiscreteWord3 }: LandingGearPositionIndicatorsProps) => {
+    const lgciu1DataValid = lgciu1DiscreteWord1.isNormalOperation();
+    const lgciu2DataValid = lgciu2DiscreteWord1.isNormalOperation();
+
+    let lgciu1GearNotUplocked: boolean;
+    let lgciu2GearNotUplocked: boolean;
+    let lgciu1GearDownlocked: boolean;
+    let lgciu2GearDownlocked: boolean;
+    let upLockFlagShown = lgciu1DataValid && lgciu2DataValid;
+    if (location === 'left') {
+        lgciu1GearNotUplocked = lgciu1DiscreteWord3.getBitValue(11);
+        lgciu2GearNotUplocked = lgciu2DiscreteWord3.getBitValue(11);
+        lgciu1GearDownlocked = lgciu1DiscreteWord1.getBitValue(23);
+        lgciu2GearDownlocked = lgciu2DiscreteWord1.getBitValue(23);
+        upLockFlagShown = upLockFlagShown && lgciu1DiscreteWord1.getBitValue(20) && lgciu1DiscreteWord1.getBitValue(20);
```

Comment:
Should be lgciu2 for the second one?

Reply:
Oh yeah, nice copy paste error. Will fix shortly